styles/css/main.css

```diff
@@ -1,3 +1,164 @@
-body {
-  background-color: black;
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+html {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  font-size: 16px;
 }
+
+.container {
+  width: 1020px;
+  margin: 0 auto;
+}
+
+.button {
+  width: 45px;
+  height: 45px;
+  background-color: #034DB0;
+  border-radius: 5px;
+  text-align: center;
+  line-height: 10px;
+  border: 1px solid white;
+  margin-right: 5px;
+  font-family: 'Roboto', sans-serif;
+  color: white;
+  -webkit-transition: 0.1s;
+  transition: 0.1s;
+}
+.button:hover {
+  background-color: #188FFF;
+  border: 1px solid white;
+  color: white;
+  cursor: pointer;
+}
+
+.button-big {
+  width: 97px;
+  height: 45px;
+  background-color: #034DB0;
+  border-radius: 5px;
+  text-align: center;
+  line-height: 10px;
+  border: 1px solid white;
+  margin-right: 5px;
+  font-family: 'Roboto', sans-serif;
+  color: white;
+}
+.button-big:hover {
+  background-color: #188FFF;
+  border: 1px solid white;
+  color: white;
+  cursor: pointer;
+}
+
+.button-big-space {
+  width: 357px;
+  height: 45px;
+  background-color: #034DB0;
+  border-radius: 5px;
+  text-align: center;
+  line-height: 10px;
+  border: 1px solid white;
+  margin-right: 5px;
+  font-family: 'Roboto', sans-serif;
+  color: white;
+}
+.button-big-space:hover {
+  background-color: #188FFF;
+  border: 1px solid white;
+  color: white;
```

Comment:
classes looks same, why you don't uses multyclasses in DOM for it. You will have one general class (button). and for button-space add properties that will be different

Reply:
> classes looks same, why you don't uses multyclasses in DOM for it. You will have one general class (button). and for button-space add properties that will be different

Thank you for your comment. I tried to fix